src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/AC5052A.scala

```diff
@@ -38,11 +38,13 @@ with Validators {
   override def validate(boxRetriever: Frs102AccountsBoxRetriever): Set[CtValidation] = {
     import boxRetriever._
 
+    val isMandatory = anyHaveValue(ac52(), ac53())
```

Comment:
Yes and no... You can look at it from two angles:
- AC52 or AC53 are not empty: It is mandatory
- AC52 and AC53 are empty: It must be empty